src/test/java/com/artipie/nuget/NugetITCase.java

```diff
@@ -106,10 +114,28 @@ void shouldPushPackage() throws Exception {
         );
     }
 
-    private void createNugetConfig() throws Exception {
-        final String url = String.format(
-            "http://host.testcontainers.internal:%d/my-nuget", this.port
+    // @checkstyle MagicNumberCheck (2 lines)
```

Comment:
@genryxy I think we may move this warning to class level, as both test methods in this class have `@Timeout` annotations